bluefog/torch/mpi_ops.cc

```diff
@@ -92,7 +92,7 @@ int DoAllreduce(::torch::Tensor tensor, ::torch::Tensor output, int average,
     auto bf_output = bf_tensor;
 
     auto enqueue_result = EnqueueTensorAllreduce(
-        bf_tensor, bf_tensor, op_name, device,
+        bf_tensor, bf_tensor, op_name, CPU_DEVICE_ID,
```

Comment:
Don't quite understand the device change here.

Reply:
See above